backend/API/appsettings.Development.json

```diff
@@ -5,7 +5,7 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "ConnectionStrings": {
-    "RookiesConnectionString": "Server=localhost\\SQLExpress;Database=TheRookiesDB;Trusted_Connection=True;"
-  }
+    "ConnectionStrings": {
+        "RookiesConnectionString": "Server=localhost\\MSSQLSERVER2019;Database=TheRookiesDB;Trusted_Connection=True;"
```

Comment:
This connection string no need to check in. You just keep your connection string on local machine - Just DEV app setting